docs/pages/versions/unversioned/sdk/blob.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Blob
+description: A web standards-compliant Blob implementation for React Native.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-blob'
+packageName: 'expo-blob'
+iconUrl: '/static/images/packages/expo-file-system.png'
+platforms: ['android', 'ios', 'web']
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { SnackInline } from '~/ui/components/Snippet';
+
+`expo-blob` provides a web standards-compliant Blob implementation for React Native that offers superior performance and reliability compared to the implementation exported from `react-native`. Unlike Blob from `react-native`, which has limitations with the `slice()` method and other Web API features, `expo-blob` provides a complete, standards-compliant implementation that works consistently across all platforms.
```

Comment:
```suggestion
`expo-blob` provides a web standards-compliant Blob implementation for React Native that offers superior performance and works consistently across all platforms. It is more reliable compared to the implementation exported from `react-native`, unlike Blob from `react-native`, which has limitations with the `slice()` method and other Web API features.
```

Last line was repetitive since the same context is shared in the first line. I'd suggest rewriting this introduction paragraph.